app/src/main/java/org/covidwatch/android/di/AppModule.kt

```diff
@@ -0,0 +1,30 @@
+package org.covidwatch.android.di
```

Comment:
What's happening in this file?

Reply:
this is where we'll declare some of our dependencies so we can later access them via the constructor or injections